app-tasks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
```

Comment:
You could install Git and then uninstall before finalizing the container image with something like:

```dockerfile

RUN set -ex \
  && buildDeps=' \
    git \
  ' \
  && apt-get update && apt-get install -y ${buildDeps} --no-install-recommends \
  && rm -rf /var/lib/apt/lists/* \
  && pip install --no-cache-dir -r /tmp/requirements.txt \
  && rm /tmp/requirements.txt \
  && apt-get purge -y --auto-remove ${buildDeps}
```